feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchViewModel.kt

```diff
@@ -0,0 +1,63 @@
+package com.hedvig.android.odyssey.search
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hedvig.android.odyssey.model.ItemType
+import com.hedvig.android.odyssey.model.SearchableClaim
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+class SearchViewModel(
+  private val getClaimEntryPoints: GetClaimEntryPoints,
+) : ViewModel() {
+  private val searchableClaims = emptyList<SearchableClaim>()
```

Comment:
This doesn't seem to be used anywhere, is this leftover or WIP in some way? Otherwise maybe _viewState cna be initiated with `results = emptyList()` inline without the need of this variable?